server/cmwell-cons/src/main/scala/ComponentConf.scala

```diff
@@ -501,6 +501,9 @@ case class BgConf(home : String, zookeeperServers : Seq[String], clusterName: St
         "-DftsService.scrollLength=100",
         "-DftsService.waitForGreen=true",
         "-DftsService.scrollTTL=3600",
+        "-Dakka.loggers.0=akka.event.slf4j.Slf4jLogger",
+        "-Dakka.loglevel=DEBUG",
```

Comment:
Please add a comment explaining that akka log level should be set to DEBUG, since we control log levels using logback.